extension/src/mcp/server.ts

```diff
@@ -308,6 +306,18 @@ const getGlossaryTermsSchema = z.object({
     .describe(
       "Optional source language code (default en-US) used as the source terminology column."
     ),
+  localGlossaryPath: z
+    .string()
+    .optional()
+    .describe(
+      "Optional absolute path to a local glossary file. If provided, the local glossary will be merged with the built-in glossary, with local terms taking precedence for duplicate entries. The file must be in TSV format with the first column as en-US (source language), the last column as Description (optional), and all columns in between as language codes. The first line must contain ISO language codes as headers."
+    ),
```

Comment:
These two new parameters must be added to the `getGlossaryTerms` tool in package.json as well, right?